### PR TITLE
Issue template files added

### PR DESCRIPTION
**Pull request by abh139**
This PR is under issue #6 and in this PR I have added only Issue template files only Please check this first and after that I will make another PR for the PR template.

### CODE REVIEW

File: Issue_template.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
```
Comment (AvineshTripathi) [inline Issue_template.yml]: I don't think there is requirement of contact details in the form

**Comment by AvineshTripathi** [comment]
Also inside the main body of `what happened ` can a contributor add images or is there any place in the form where we can add screenshot if required

File: Issue_template.yml (hunk 1 above)
Comment (abh139) [reply Issue_template.yml]: Yes I updated that

**Comment by abh139** [comment]
> Also inside the main body of `what happened ` can a contributor add images or is there any place in the form where we can add screenshot if required

You can add screenshot in the screenshot section

**Comment by AvineshTripathi** [comment]
if you dont mind can you squash the commits, it would be easier to review it once done 
p.s. you can refer this https://www.git-tower.com/learn/git/faq/git-squash

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Paths for all the issue templates won't work.
Please move all the files to `.github/ISSUE_TEMPLATE/*.yaml`

Also note that we cannot have spaces in files or directory names. Use hyphen (`-`) instead.

**Comment by anubha-v-ardhan** [comment]
I've made another issue #14 for PR templates and changed #6 for issue templates only.
This PR will close #6.

File: .github/Issue_Template/Feature_Update.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: 💡 Feature Request
+description: Have a new idea/feature for LinkFree? Please suggest!
```
Comment (SuperAayush) [inline .github/Issue_Template/Feature_Update.yaml:2]: Please change the name of the project from Linkfree to Kubesimplify.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by SuperAayush — CHANGES_REQUESTED** [review]

File: .github/Issue_Template/Feature_Update.yaml (hunk 2 above)
Comment (abh139) [reply .github/Issue_Template/Feature_Update.yaml:2]: Please check the latest commits

**Comment by AvineshTripathi** [comment]
@anubha-v-ardhan do you think we should close this pr and do another one it's very difficult to review this PR

**Pull request closed without merging**